internal/stacktrace/trie.go

```diff
@@ -9,139 +9,12 @@ import (
 	"sync"
```

Comment:
This should be removed, from the multiple warnings below.

Also, I think we should update `CODEOWNERS` to ensure only LP/guild's approval is required.